spiketoolkit/preprocessing/filterrecording.py

```diff
@@ -46,8 +52,9 @@ def get_traces(self, channel_ids=None, start_frame=None, end_frame=None):
                 else:
                     end0 = self._chunk_size
                 chan_idx = [self.get_channel_ids().index(chan) for chan in channel_ids]
-                filtered_chunk_list.append(filtered_chunk0[chan_idx, start0:end0])
-            filtered_chunk =  np.concatenate(filtered_chunk_list, axis=1)
+                filtered_chunk[:, start0:end0] = filtered_chunk0[chan_idx, start0:end0]
+                # filtered_chunk_list.append(filtered_chunk0[chan_idx, start0:end0])
+            #�filtered_chunk =  np.concatenate(filtered_chunk_list, axis=1)
         else:
             filtered_chunk = self.filter_chunk(start_frame=0, end_frame=self._recording.get_num_frames())
```

Comment:
This should be `filtered_chunk = self.filter_chunk(start_frame=0, end_frame=self._recording.get_num_frames())[channel_ids, start_frame:end_frame]` for consistency

Reply:
Why not 
```filtered_chunk = self.filter_chunk(start_frame=start_frame:end_frame, end_frame=end_frame)[channel_ids, :]```

Reply:
Also good

Reply:
smater you want to say.